issue_processing/jira_issue_processing.py

```diff
@@ -566,20 +564,74 @@ def print_to_disk(issues, results_folder):
     lines = []
     for issue in issues:
         log.info("Current issue '{}'".format(issue["externalId"]))
-        lines.append((issue["author"]["name"],
-                      issue["author"]["email"],
-                      issue["externalId"],
-                      issue["creationDate"],
-                      issue["externalId"],
-                      issue["type"]))
+        lines.append((
+            issue["externalId"],
+            issue["title"],
+            json.dumps(issue["type_list"]),
+            issue["state_new"],
+            json.dumps(issue["resolution_list"]),
+            issue["creationDate"],
+            issue["resolveDate"],
+            json.dumps(issue["components"]),
+            "created",  ## event.name
+            issue["author"]["name"],
+            issue["author"]["email"],
+            issue["creationDate"],
+            "open",  ## default state when created
+            json.dumps(["unresolved"])  ## default resolution when created
+        ))
+
+        lines.append((
```

Comment:
Just one comment here: Why do you call `lines.append` twice with almost the same parameters?
At first glance, I was confused and did not see that there is a little difference regarding `"created"` and `"commented"`. After talking to @clhunsen, I now  understand what's going on here. However, it would be helpful to have a comment in the code that explains why this event is added twice and what the difference is. So, @fehnkera, could you please add some documentation for that? Thanks!